test/CMakeLists.txt

```diff
@@ -67,9 +73,6 @@ set(WINDOW_SRC
     Window/WindowBase.test.cpp
 )
 sfml_add_test(test-sfml-window "${WINDOW_SRC}" SFML::Window)
-if(SFML_RUN_DISPLAY_TESTS)
-    target_compile_definitions(test-sfml-window PRIVATE SFML_RUN_DISPLAY_TESTS)
-endif()
```

Comment:
Is this no longer needed as it's being set on `sfml-test-main`

Reply:
That's right